### ISSUE TITLE
Default .golangci.reference.yml config is invalid according to what golangci-lint says

### ISSUE THREAD

**Issue by mateusz-szczyrzyca** [golangci/golangci-lint#3627]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc.).
- [X] Yes, I've tried with the standalone linter if available (e.g., gocritic, go vet, etc.). (https://golangci-lint.run/usage/linters/)

### Description of the problem

Reference config file is broken (according to golangci-lint) - I just download file, rename as `.golangci.yml` inside a go project directory and when I'm trying to use `golangci-lint` I always got:

```ERRO Can't read config: can't validate config: error in govet config: enable-all and disable-all can't be combined```

Reference config: https://raw.githubusercontent.com/golangci/golangci-lint/master/.golangci.reference.yml

No matter what `golangci-lint` command I try to use - it's impossible to check even version.
If I run `golangci-lint` without this file (outside the project), it works fine.

BTW: GitHub says `Comment is too long (maximum is 65536 characters)` when I tried to paste entire config as you asked.

### Version of golangci-lint

<details>

```console
golangci-lint has version 1.51.2 built from 3e8facb4 on 2023-02-19T21:43:54Z
```

</details>

### Configuration file

<details>
It's currently config-reference: https://raw.githubusercontent.com/golangci/golangci-lint/master/.golangci.reference.yml - can't paste it here as GitHub complains about body length.
</details>

### Go environment

<details>

```console
$ go version && go env
go version go1.20.1 darwin/arm64
GO111MODULE=""
GOARCH="arm64"
GOBIN=""
GOCACHE="/Users/mateusz/Library/Caches/go-build"
GOENV="/Users/mateusz/Library/Application Support/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="arm64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="/Users/mateusz/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/mateusz/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/Users/mateusz/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/Users/mateusz/go/pkg/tool/darwin_arm64"
GOVCS=""
GOVERSION="go1.20.1"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/dev/null"
GOWORK=""
CGO_CFLAGS="-O2 -g"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-O2 -g"
CGO_FFLAGS="-O2 -g"
CGO_LDFLAGS="-O2 -g"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -arch arm64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/qx/2xcls5j57rd4ycjxcrt62ctm0000gn/T/go-build74666405=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
(base)  ✘ mat@workstation-4  ~/golang-projects/footballdata   master  golangci-lint cache clean
ERRO Can't read config: can't validate config: error in govet config: enable-all and disable-all can't be combined
(base)  ✘ mat@workstation-4  ~/golang-projects/footballdata   master  golangci-lint run -v
INFO [config_reader] Config search paths: [./ /Users/mateusz/golang-projects/footballdata /Users/mateusz/golang-projects /Users/mateusz /Users /]
INFO [config_reader] Used config file .golangci.yml
ERRO Can't read config: can't validate config: error in govet config: enable-all and disable-all can't be combined
```

</details>

### Code example or link to a public repository

<details>

```go
Checked on 3 different projects and results were the same
```

</details>

**Comment by ldez** [comment]
Hello,

https://github.com/golangci/golangci-lint/blob/610a2bd199a376bdbc37431ce2ba52d85f0ee307/.golangci.reference.yml#L1-L5


the configuration reference is not an example, so yes the configuration is not valid and it's expected.
